src/components/account/CreateAccount.tsx

```diff
@@ -23,7 +22,10 @@ const CreateAccount = () => {
     string[] | null
   >(null)
 
-  const createAccount = useCreateAccountMutation()
+  const createAccount = useCreateAccountMutation({
+    setMessage,
+    successMessage: 'Account successfully created!',
```

Comment:
I actually should have mentioned this but, at least for ADDI, we found that we always have the same success message for a mutation so we don't even pass it, we just have it hardcoded into the mutation. Up to you to determine if you think that will hold true in your case as well.